src/app/[locale]/(logged-in)/data-custodian/profile/users/[id]/projects/[projectId]/[subTabId]/components/SubsTabContents/SubTabsContents.tsx

```diff
@@ -0,0 +1,52 @@
+"use client";
+
+import { useStore } from "@/data/store";
+import { notFound } from "next/navigation";
+import UserAffiliations from "../../../../../../../components/UserAffiliations";
+import UserCustodianOrgInfo from "../../../../../../../components/UserCustodianOrgInfo";
+import UserHistory from "../../../../../../../components/UserHistory";
+import UserIdentity from "../../../../../../../components/UserIdentity";
+import UserProjects from "../../../../../../../components/UserProjects";
+import UserTrainingAccreditations from "../../../../../../../components/UserTrainingAccreditations";
+import { UserSubTabs } from "../../../../../../../consts/tabs";
```

Comment:
Update all these

Reply:
I decided against it as it would be `@/app/[locale]/...` which isnt what you're suppose to do.. we need to rearrange these sections and move the components to the current folder instead of the higher level

Reply:
Gotcha